infra/us-east-1/prod/eks/blue/eks-addons/terragrunt.hcl

```diff
@@ -21,7 +21,7 @@ include {
 }
 
 terraform {
-  source = "github.com/input-output-hk/sc-dev-platform.git//infra/modules/eks/addons"
+  source = "../../../../../modules/eks/addons"
```

Comment:
Why were we re-fetching from GH?

Reply:
This was something we had considered but that didn't work well, and were moving away from. We had already reverted it in most places, so it's good that it's reverted here as well.